--- a/vendor/jwage/easy-csv/lib/EasyCSV/Reader.php
+++ b/vendor/jwage/easy-csv/lib/EasyCSV/Reader.php
@@ -21,12 +21,17 @@
         $this->init();
         return $this->headers;
     }
-
+
     public function getRow()
     {
         $this->init();
-        if (($row = fgetcsv($this->handle, 1000, $this->delimiter, $this->enclosure)) !== false) {
+        if (($row = fgetcsv($this->handle, 0, $this->delimiter, $this->enclosure)) !== false) {
             $this->line++;
+            $tmpArray = array();
+            foreach ($row as $key => $value) {
+                $tmpArray[$key] = iconv("ISO-8859-1", "UTF-8//TRANSLIT", $value);
+            }
+            $row = $tmpArray;
             return $this->headers ? array_combine($this->headers, $row) : $row;
         } else {
             return false;
@@ -46,7 +51,7 @@
     {
         return $this->line;
     }
-
+
     protected function init()
     {
         if (true === $this->init) {
